js/rpg_managers/DataManager.js

```diff
@@ -41,6 +41,8 @@ DataManager._globalId       = 'RPGMV';
 DataManager._lastAccessedId = 1;
 DataManager._errorUrl       = null;
 
+DataManager.imageReservationId  = Utils.generateRuntimeId();
```

Comment:
DataManager don't have responsibility of cancelling reservation.
It's ugly but We cannot move loading methods to the Scene because of compatibility.